kustomize/gatewayapi/patch.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: flagger
+spec:
+  template:
+    spec:
+      containers:
+        - name: flagger
+          image: aryan9600/flagger
+          args:
+            - -log-level=debug
+            - -include-label-prefix=app.kubernetes.io
+            - -mesh-provider=gatewayapi
```

I suggest we this naming everywhere:
 
```suggestion
            - -mesh-provider=gateway:v1alpha2
```